### PR TITLE
Nextcloud and additional storage

### PR DESCRIPTION
**Pull request by stilobique**
Hello,
I write a quick update about the nextcloud use and about the additional storage use -in French only, look to make the translation this week, maybe later.

### CODE REVIEW

File: pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md [hunk 1]
```diff
@@ -9,28 +9,31 @@ routes:
...
+    - [Ajouter de l'espace à Nextcloud](#ajout-espace)
+    - [Partager un dossier entre nextcloud et une application](#partager-un-dossier-entre-nextcloud-et-une-application)
+    - [Nextcloud et Cloudlare](#nextcloud-et-cloudflare)
+ - [Applications tierce](#application-tierce)
+    - [A propos de keeweb](#à-propos-de-keeweb)
  - [Liens utiles](#liensutiles)
 
-Nextcloud est un service d'hébergement de fichiers, de nombreuses applications peuvent être installées afin de lui offrir de nouvelles fonctionnalités tel qu'un agenda, un répertoire de contacts, des notes et pleins d'autres possibles (vous pouvez trouver quelques applications dans la partie [applications tiers](#AppsTiers) mais il en existe une multitude suivant vos besoins).
+Nextcloud est un service d'hébergement de fichiers, de nombreuses applications peuvent être installées afin de lui offrir de nouvelles fonctionnalités tel qu'un agenda, un répertoire de contacts, des notes et pleins d'autres possibles (vous pouvez trouver quelques applications dans la partie [applications tiers](#application-tierce) mais il en existe une multitude suivant vos besoins).
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md:22]:
```suggestion
Nextcloud est un service d'hébergement de fichiers. De nombreuses applications peuvent être installées afin d'offrir à Nextcloud de nouvelles fonctionnalités, telles qu'un agenda, un répertoire de contacts, des notes et plein d'autres possibles (vous pouvez trouver quelques applications dans la partie [applications tiers](#applications-tierces) mais il en existe une multitude suivant vos besoins).
```

File: pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md [hunk 2]
```diff
@@ -142,6 +145,31 @@ sudo -u nextcloud php7.3 occ files:scan --all
 
 C'est terminé. À présent testez si tout va bien, essayez de vous connecter à votre instance Nextcloud, envoyer un fichier, vérifiez sa bonne synchronisation.
 
+### Partager un dossier entre Nextcloud et une application
+Il est relativement simple de monter des dossiers accessibles depuis nextcloud en lecture/écriture et de les
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md:149]:
```suggestion
Il est relativement simple de monter des dossiers accessibles depuis Nextcloud en lecture/écriture et de les 
```

File: pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md [hunk 3]
```diff
@@ -142,6 +145,31 @@ sudo -u nextcloud php7.3 occ files:scan --all
 
 C'est terminé. À présent testez si tout va bien, essayez de vous connecter à votre instance Nextcloud, envoyer un fichier, vérifiez sa bonne synchronisation.
 
+### Partager un dossier entre Nextcloud et une application
+Il est relativement simple de monter des dossiers accessibles depuis nextcloud en lecture/écriture et de les 
+partager avec d'autres applications (par exemple [Jellyfin](app_jellyfin), [Funkwhale](app_funkwhale), [Transmission](app_transmission), ...)
+
+Il Vous fait commencer par monter un dossier qui sera disponible pour votre application (je prends jellyfin dans mon
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md:152]:
```suggestion
Il vous faut commencer par monter un dossier qui sera disponible pour votre application (je prends Jellyfin dans mon 
```

File: pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md [hunk 4]
```diff
@@ -165,6 +193,11 @@ Les options à désactiver (Off) sont :
 
 Sauvegarder et nettoyer vos caches (Cloudflare, navigateur...) et le tour est joué.
 
+# Application Tierce
+Certaine application son disponible directement depuis Nextcloud.
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md:197]:
```suggestion
# Applications tierces
Certaines applications sont disponibles directement depuis Nextcloud.
```

File: pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md [hunk 5]
```diff
@@ -165,6 +193,11 @@ Les options à désactiver (Off) sont :
...
+![image](image://nextcloud_menu_parameter.jpg)
+
+
 ## À propos de Keeweb
 
 L'application Keeweb sur le catalogue de nextcloud - [apps.nextcloud.com/keeweb](https://apps.nextcloud.com/apps/keeweb)
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md:203]:
```suggestion
L'application Keeweb sur le catalogue de Nextcloud - [apps.nextcloud.com/keeweb](https://apps.nextcloud.com/apps/keeweb)
```

File: pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md [hunk 6]
```diff
@@ -9,28 +9,31 @@ routes:
...
 ![logo de Nextcloud](image://nextcloud_logo.png)
 
- - [Découverte de l'environnement de Nextcloud](#EnvironnementNextcloud)
- - [Logiciels Clients pour mobile et ordinateur](#LogicielsClients)
- - [Manipulations & Problèmes rencontrés utiles](#ManipulationsUtiles)
-    - [Ajouter de l'espace à Nextcloud](#AjoutEspace)
- - [Applications tiers](#AppsTiers)
+ - [Découverte de l'environnement de Nextcloud](#environnement-nextcloud)
+ - [Logiciels Clients pour mobile et ordinateur](#logiciels-clients)
+ - [Manipulations & Problèmes rencontrés utiles](#manipulations-utiles)
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md:14]:
```suggestion
 - [Logiciels clients pour mobile et ordinateur](#logiciels-clients)
 - [Manipulations & problèmes rencontrés utiles](#manipulations-utiles)
```

File: pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md [hunk 7]
```diff
@@ -9,28 +9,31 @@ routes:
...
+    - [Ajouter de l'espace à Nextcloud](#ajout-espace)
+    - [Partager un dossier entre nextcloud et une application](#partager-un-dossier-entre-nextcloud-et-une-application)
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md:16]:
```suggestion
    - [Partager un dossier entre Nextcloud et une application](#partager-un-dossier-entre-nextcloud-et-une-application)
```

File: pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md [hunk 8]
```diff
@@ -9,28 +9,31 @@ routes:
...
+    - [Nextcloud et Cloudlare](#nextcloud-et-cloudflare)
+ - [Applications tierce](#application-tierce)
+    - [A propos de keeweb](#à-propos-de-keeweb)
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/nextcloud/app_nextcloud.fr.md:19]:
```suggestion
 - [Applications tierces](#applications-tierces)
    - [À propos de Keeweb](#à-propos-de-keeweb)
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]
Fix typos

**Review by ericgaspar — APPROVED** [review]

**Comment by stilobique** [comment]
First update about the documentation 🥳 .
Thanks @ericgaspar for your feedback.

**Pull request merged**